sql/log_event.cc

```diff
@@ -9682,6 +9682,20 @@ search_key_in_table(TABLE *table, MY_BITMAP *bi_cols, uint key_type)
   uint res= MAX_KEY;
   uint key;
 
+  /*
```

Comment:
Tests for primary-secondary scenairo should be added. But, we dont see any cases.

Reply:
This logic belongs to master slave synchronization logic, and MTR cannot test it.
It has been verified in our test server.

Reply:
To add test cases